lib/govuk_publishing_components/presenters/page_with_step_by_step_navigation.rb

```diff
@@ -81,12 +95,29 @@ def active_step_by_step
         @active_step_navs.first
       end
 
+      def secondary_step_by_step?
+        secondary_step_by_steps.any? && step_navs_combined_list.count === 0
+      end
+
+      def secondary_step_by_step
+        secondary_step_by_steps.first
+      end
+
+      def show_secondary_step_by_step?
+        secondary_step_by_step? && secondary_step_by_steps.count === 1
+      end
+
+      def show_related_links_for_secondary_step_by_steps?
+        secondary_step_by_step? && secondary_step_by_steps.count < 5
```

Comment:
As with `show_secondary_step_by_step?` I think this might be more readable if it's expanded out, e.g.

```
!primary_step_by_step? && secondary_step_by_steps.any? && secondary_step_by_steps.count < 5
```